modules/create-storefront-x/template/venia/modules/demo-venia/organisms/Footer.vue

```diff
@@ -6,160 +6,50 @@
+                        <Link :to="localePath(t(pageLink.path))" color="gray">
+                          {{ t(pageLink.label) }}
                         </Link>
                       </li>
                     </ul>
                   </template>
                 </Accordion>
               </div>
-
-              <div>
-                <span class="border-gray-50 font-semibold text-gray-500 hidden sm:block">
-                  {{ t('Account') }}
+            </div>
+            <div
+              class="grid whitespace-nowrap space-y-4 sm:space-y-0 sm:col-span-3 sm:grid-cols-3 sm:gap-x-40 hidden md:grid"
+            >
+              <div v-for="(pageLinkCategory, index) in pageLinksByCategory" :key="index">
+                <span class="border-gray-50 font-semibold text-gray-500">
+                  {{ t(pageLinkCategory.name) }}
                 </span>
-                <Accordion
-                  :icon-class="{ 'sm:hidden': true, 'display-block': true, 'ml-2': true, 'cursor-pointer': true }"
-                  :heading-class="[
-                    'border-gray-50',
-                    'sm:justify-start',
-                    'justify-center',
-                    'sm:hidden',
-                    'block',
-                    'font-semibold',
-                    'text-gray-500',
-                  ]"
-                  :content-class="['text-center', 'sm:text-left', 'mb-12', 'sm:mb-4']"
-                  :open="true"
-                >
-                  <template #heading>
-                    {{ t('Account') }}
-                  </template>
-
-                  <template #default>
-                    <ul role="list" class="mt-6 pl-0 space-y-6 list-none">
-                      <li class="text-sm">
-                        <Link :to="t('sign-in')" color="gray">
-                          {{ t('Sign in') }}
-                        </Link>
-                      </li>
-
-                      <li class="text-sm">
-                        <Link :to="t('sign-up')" color="gray">
-                          {{ t('Sign up') }}
-                        </Link>
-                      </li>
 
-                      <li class="text-sm">
-                        <Link :to="t('account')" color="gray">
-                          {{ t('Settings') }}
-                        </Link>
-                      </li>
-
-                      <li class="text-sm">
-                        <Link :to="t('account/orders')" color="gray">
-                          {{ t('My orders') }}
-                        </Link>
-                      </li>
-                    </ul>
-                  </template>
-                </Accordion>
+                <ul role="list" class="mt-6 pl-0 space-y-6 list-none">
+                  <li v-for="(pageLink, indexChild) in pageLinkCategory.children" :key="indexChild" class="text-sm">
+                    <Link :to="localePath(t(pageLink.path))" color="gray" class="hover:underline">
```

Comment:
I dont like that you are translating every path (this might cause a bug when coliding with global translations). I would rather see `t` being used directly in the `pageLinksByCategory` array and here will be only `localePath`